### PR TITLE
fix(CAD): pass entrypoint thru to /pair

### PR DESCRIPTION
**Pull request by chenba**
Because:
 - the entrypoint param from CAD to /pair was hardcoded

This commit:
 - pass the (valid) entrypoint param through to /pair

Fixes FXA-5990

### CODE REVIEW

File: packages/fxa-content-server/app/tests/spec/views/connect_another_device.js [hunk 1]
```diff
@@ -120,37 +120,79 @@ describe('views/connect_another_device', () => {
...
-            sinon.spy(view, 'navigate');
-
-            windowMock.navigator.userAgent =
-              'Mozilla/5.0 (Macintosh; Intel Mac OS X 10.12; rv:55.0) Gecko/20100101 Firefox/55.0';
-
-            return view.render().then(() => {
-              view.afterVisible();
-            });
-          });
+    [
```
Comment (dschom) [inline packages/fxa-content-server/app/tests/spec/views/connect_another_device.js:123]: Should all 7 of these entry points be checked? https://github.com/mozilla/fxa/blob/main/packages/fxa-content-server/app/scripts/lib/constants.js#L168

BTW, I checked them manually. The entry point param is propagated through correctly in all cases. When entrypoint=fxa_discoverability_native, the code immediately redirects to /pair/unsupported view however.

**Review by dschom — APPROVED** [review]
LGTM! I manually tested as well and the changes checked out.

File: packages/fxa-content-server/app/scripts/views/connect_another_device.js [hunk 2]
```diff
@@ -14,18 +14,23 @@ import FlowEventsMixin from './mixins/flow-events-mixin';
...
-} from '../lib/constants';
+import Constants from '../lib/constants';
 import MarketingMixin from './mixins/marketing-mixin';
 import MarketingSnippet from './marketing_snippet';
 import SyncAuthMixin from './mixins/sync-auth-mixin';
 import Template from 'templates/connect_another_device.mustache';
 import UserAgentMixin from '../lib/user-agent-mixin';
 import VerificationReasonMixin from './mixins/verification-reason-mixin';
 
+const entrypoints = Object.keys(Constants)
```
[the remark this answers is not in the document]
Comment (chenba) [reply packages/fxa-content-server/app/scripts/views/connect_another_device.js:25]: I think we mostly use 'entrypoint' in the content-server.  There's also https://github.com/mozilla/fxa/blob/469242cf89b7dfd7ed96ff3fbe913dc3a386a7ba/packages/fxa-content-server/app/scripts/models/reliers/relier.js#L172-L176

File: packages/fxa-content-server/app/tests/spec/views/connect_another_device.js (hunk 1 above)
Comment (chenba) [reply packages/fxa-content-server/app/tests/spec/views/connect_another_device.js:123]: > Should all 7 of these entry points be checked?

I'm not sure.  I'm not sure how the previous four was picked

> When entrypoint=fxa_discoverability_native, the code immediately redirects to /pair/unsupported view however.

That sounds wrong.  But when I tried it by selecting "Connect another device..." menu item on the Firefox FxA menu, it worked just fine.

**Pull request merged**